sycl/test/config/config.cpp

```diff
@@ -0,0 +1,26 @@
+//==---- config.cpp --------------------------------------------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// RUN: %clangxx -g -O0 -fsycl %s -o %t.out
+// RUN: echo "SYCL_PRINT_EXECUTION_GRAPH=always" > %t.cfg
+// RUN: env SYCL_CONFIG_FILE_NAME=%t.cfg %t.out
+// RUN: ls | grep dot
+// RUN: rm *.dot
+// RUN: env SYCL_PRINT_EXECUTION_GRAPH=always %t.out
+// RUN: ls | grep dot
```

Comment:
Are you sure that this works on Windows if you don't have GNU coreutils installed?

Reply:
I've reused this idea from clang LIT tests so I expect it works on the same configurations as clang LIT tests work.